### DISCUSSION TITLE
Label Border Dimensions

### DISCUSSION THREAD

**Discussion by heschdy** [Part-DB/Part-DB-server#660] [General]
Hi,

is it possible to configure the border width for Labels? Im loosing on small Labels (38x25mm) a lot of space because on the top and bottom border.

![image](https://github.com/user-attachments/assets/1f884c95-b38a-46f2-958f-cac59b91e798)

Thanks
Ralf

**Comment by jbtronics** [comment]
You can customize the CSS that creates the page borders, via the "Additional CSS" option for the labels: 

```css
@page {
    margin: 12px 6px;
}
```

Use the values that fit your needs.

**Reply by heschdy** [reply]
Thanks, works like a charm!
